commands/ImportController.php

```diff
@@ -33,6 +33,19 @@
  */
 class ImportController extends Controller
 {
+    /**
+     * @var string[]
+     */
+    public static $objectTypesMap = [
```

Comment:
This map could be moved to `ObjectKeyHelper` itself.

Reply:
No, map is needed exclusively for import. I think a helper should not know about this.